components/sandbox/toolchain-host-operator/production/stone-prd-host1/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ resources:
 - ../../base
 - toolchainconfig.yaml
 - space-provisioner-configs.yaml
+- miner-detector-cronjob.yaml
```

Comment:
We only need this on the private production cluster? Should we run this on the public cluster also?

Reply:
> We only need this on the private production cluster? 

TBH we are not even 100% sure we still need to run this anymore. But just for consistency we decided to keep it there and just move the configs for the cluster that has this tool enabled.  

> Should we run this on the public cluster also?

Isn't this the "public" cluster , I mean the one that is running our production workloads ? You can verify and see that we are already running this job in the `toolchain-host-operator` namespace.

Reply:
sorry, my bad, I hallucinated stone-prod-p01 :) You are correct. Resolving this comment.